client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx

```diff
@@ -136,14 +137,13 @@ class PendingGSuiteTosNotice extends React.PureComponent {
 			<>
 				<p>
 					{ translate(
-						'Your mailbox {{strong}}%(emails)s{{/strong}} is almost ready! Complete the setup to activate your new email address.',
-						'Your mailboxes {{strong}}%(emails)s{{/strong}} are almost ready! Complete the setup to activate your new email addresses.',
+						'Your new user {{strong}}%(emails)s{{/strong}} is ready! Complete the setup in order to activate it.',
+						'Your new users {{strong}}%(emails)s{{/strong}} are ready! Complete the setup in order to activate them.',
 						{
 							count: users.length,
 							args: { emails },
 							comment:
-								'%(emails)s will be a list of email addresses separated by a comma, ' +
-								'e.g. test@example.com, test2@example.com',
+								"%(emails)s is a list of email addresses separated by a comma (e.g. 'one@example.com, two@example.com')",
```

Comment:
Might this be clearer if we state it as:
> %(emails)s is a list of email addresses separated by commas (e.g. 'one@example.com, two@example.com')

Not sure if that actually has any material benefit, so feel free to skip this to keep the existing comment closer to the existing one.